packages/registry/profiles/logitech/logitech-mx-ink.json

```diff
@@ -0,0 +1,31 @@
+{
+  "profileId": "logitech-mx-ink",
+  "fallbackProfileIds": [
+    "generic-trigger-squeeze-thumbstick"
```

Comment:
Is this the right fallback ID? It's hard for me to tell the exact capabilities from the marketing materials out there, but at first blush this seems more like it would fall back to `generic-trigger` or `generic-button`? Also, down in the components section you list "touch-pad", which seems to conflict with the "-thumbstick" in this profileID.